src/wasm2js.h

```diff
@@ -2419,41 +2423,40 @@ void Wasm2JSGlue::emitPostES6() {
   //
   // Note that the translation here expects that the lower values of this memory
   // can be used for conversions, so make sure there's at least one page.
-  {
-    auto pages = wasm.memory.initial == 0 ? 1 : wasm.memory.initial.addr;
```

Comment:
I suspect this was because creating an ArrayBuffer (or a typed array or whatever we create here) with a size of 0 is not valid. It would be good to verify this works, as I'm not sure we have a test for that specifically.

Reply:
Testing this on node, both old and new, it seems ok... so maybe the worry was irrelevant?

Reply:
Seems to work in chrome too.